### PR TITLE
Minor fixes to the Portfolio tests

### PR DESCRIPTION
**Pull request by tchaffee**
#### Pre-Submission Checklist

- [x] Your pull request targets the `master` branch of freeCodeCamp/testable-projects-fcc
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] Your changes have been tested either locally or using a newly created CDN based on your fork's testable-projects-fcc/build/bundle.js file
- [x] Working changes have been added to the build by running `npm run build`

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.

- [ ] Closes currently open issue: Closes #XXX

#### Description
- Added the tech stack requirement.
- We don't need to use FCC_Global if we import chai. With this small change there are now zero ESLint warnings in this file.
- Changed the numbering of the tests to be dynamic. This allows more easily adding new tests without having to edit the numbers. It also justifies using string template literals (interpolated string literals).
- Replaced 'for loop' logic with safer Array.prototype.some() logic for the navbar scrolling tests.
- Replaced 'for loop' logic with safer filtering to detect media queries.

### CODE REVIEW

**Comment by tchaffee** [comment]
@tbushman Please review https://github.com/freeCodeCamp/testable-projects-fcc/pull/165 before this one. If there are any changes there I should make those changes and rebase this PR before you review. Thanks!

File: src/project-tests/portfolio-tests.js [hunk 1]
```diff
@@ -1,133 +1,241 @@
...
-          var distanceFromBottom = bottomPositionY - window.scrollY;
-
-          if (distanceFromBottom) {
-            // if distance from bottom is not 0, clicking a link made it move
-            FCC_Global.assert.ok(true);
-            window.scroll(0, 0);
-            return;
-          }
+        didScroll = links.some(link => {
+          console.log('clicking link...');
```
Comment (tbushman) [inline src/project-tests/portfolio-tests.js]: Did you want to keep these console.logs ?

Comment (tchaffee) [inline src/project-tests/portfolio-tests.js]: Good catch. Removing these now.

**Comment by tchaffee** [comment]
This should be good to merge now. I made one other small change: moved the Technology Stack requirements to their own section. I tested everything again too.

**Pull request merged**